external/fv3fit/fv3fit/emulation/data/load.py

```diff
@@ -27,7 +27,7 @@ def nc_files_to_tf_dataset(
     """
 
     transform = compose_left(*[open_netcdf_dataset, convert])
-    return seq_to_tfdataset(files, transform).unbatch().prefetch(tf.data.AUTOTUNE)
+    return seq_to_tfdataset(files, transform).prefetch(tf.data.AUTOTUNE)
```

Comment:
Double-checking this: I don't see any changes to calls in the non-testing code of `nc_files_to_tf_dataset`. Is this an issue / have I overlooked one?

Reply:
There are changes. easy to see in commit view: https://github.com/ai2cm/fv3net/pull/1734/commits/124c7b3e4813dcabd4e9aab11b33440715ae52e7

Reply:
and we have good coverage here.